### PR TITLE
build,refactor,fix: migrated to newer dart,flutter version; fixed cod…

### PR DESCRIPTION
**Pull request by dk-a-dev**
## Summary
**build**: migrated to flutter version 3.13.0 stable and also added null safety checks
**fix and refactor**: all the libraries were migrated to null-safe versions and two were changed to support existing features

[This is the Issue Title](https://github.com/IEEE-VIT/hacktoberfest-flutter/issues/35)

## Major Changes made
- migrated the project completely to new versions of flutter
- slimy_card library changed with flutter_slimy_card as prev one was breaking and is not supporting newer dart versions
- fixed errors of the http package in the project
- changed buttons which were deprecated.
- code optimization along with refactoring to support new versions and improve code quality.

### CODE REVIEW

**Review by mintbomb27 — COMMENTED** [review]
Great job on the refactor! Any particular reason to remove the `.github` folder? The `ISSUE_TEMPLATE` and `PULL_REQUEST_TEMPLATE` does add a proper format to the issues and PRs created. Recommend adding it back.

**Comment by dk-a-dev** [comment]
> Great job on the refactor! Any particular reason to remove the `.github` folder? The `ISSUE_TEMPLATE` and `PULL_REQUEST_TEMPLATE` does add a proper format to the issues and PRs created. Recommend adding it back.

Added it back please re-review

File: .github/workflows/flutter-ci.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches:
+      - master
+
+
+env:
+  flutter_version: "1.20.4"
```
Comment (mintbomb27) [inline .github/workflows/flutter-ci.yml:13]: Please change the version here to the required version. The CI isn't passing.

**Review by mintbomb27 — CHANGES_REQUESTED** [review]

**Review by mintbomb27 — APPROVED** [review]
LGTM!

**Pull request merged**